testhelper/destination/transformer.go

```diff
@@ -21,15 +21,15 @@ func SetupTransformer(pool *dockertest.Pool, d cleaner) (*TransformerResource, e
 	// pulls an image first to make sure we don't have an old cached version locally,
 	// then it creates a container based on it and runs it
 	err := pool.Client.PullImage(docker.PullImageOptions{
-		Repository: "rudderlabs/rudder-transformer",
-		Tag:        "latest",
+		Repository: "rudderlabs/develop-rudder-transformer",
```

Comment:
will this get reverted post the transformer PR merge ?

Reply:
Yes its added now for testing purpose